dynawo/sources/Models/CPP/ModelNetwork/test/TestLoad.cpp

```diff
@@ -572,9 +585,9 @@ TEST(ModelsModelNetwork, ModelNetworkLoadJt) {
   ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[0], 22.155644370746373);
   ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[1], -153.29392144688848);
   ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[2], -87.596526541079143);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[3],  8.4895038000592429);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[4],  2.3388791341676249);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[5],  1.3365023623815);
+  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[3], 8.4895038000592429);
+  ASSERT_DOUBLE_EQUALS_DYNAWO(smj.Ax_[4], 5.2397892818982124);
```

Comment:
It appears for this test that the original test was wrong. Wrong indexes were used in the evallJt function because model was not initialized. Model is initialized in new version